.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -3,4 +3,4 @@
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
     <option name="PREFERRED_PROJECT_CODE_STYLE" value="Project copy" />
   </state>
-</component>
+</component>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done